src/command_modules/azure-cli-dls/azure/cli/command_modules/dls/_params.py

```diff
@@ -0,0 +1,64 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from azure.cli.core.commands import \
+    (register_cli_argument, CliArgumentType)
+
+from azure.cli.core.commands.parameters import \
+    (tags_type, get_resource_name_completion_list, resource_group_name_type, enum_choice_list)
+
+from azure.cli.command_modules.dls._validators import validate_resource_group_name
+from azure.mgmt.datalake.store.models.data_lake_store_account_management_client_enums \
+    import (FirewallState,
+            TrustedIdProviderState,
+            TierType,
+            FirewallAllowAzureIpsState)
+
+from azure.mgmt.datalake.store.models import (EncryptionConfigType)
+
+# ARGUMENT DEFINITIONS
+# pylint: disable=line-too-long
+datalake_store_name_type = CliArgumentType(help='Name of the Data Lake Store account.', options_list=('--account_name',), completer=get_resource_name_completion_list('Microsoft.DataLakeStore/accounts'), id_part='name')
+
+# PARAMETER REGISTRATIONS
+# global params
+register_cli_argument('dls', 'account_name', datalake_store_name_type, options_list=('--account', '-n'))
+register_cli_argument('dls', 'top', help='Maximum number of items to return.', type=int)
+register_cli_argument('dls', 'skip', help='The number of items to skip over before returning elements. Optional.', type=int)
+register_cli_argument('dls', 'count', help='The Boolean value of true or false to request a count of the matching resources included with the resources in the response, e.g. Categories?$count=true. Optional.', type=bool)
```

Comment:
You don't need to add `Optional.` to the help strings. The CLI adds `[Required]` to arguments that are required automatically.